README.md

```diff
@@ -473,22 +485,193 @@ When the scanner successfully decodes a `TransactionRequest` it will call its de
 </manifest>
 ```
 
+# Websocket
+
+After we are able to create a `TransactionRequest`, consume a `TransactionRequest`, and then approve or reject a `TransactionConsumption`.
+Now, we will learn how to use the web socket client for listening for the `TransactionConsumption` events.
+This section describes the use of the `OMGSocketClient` in order to listen for events.
+
+## Websocket Initialization
+
+Similarly to the `OMGAPIClient`, the `OMGSocketClient` needs to be first initialized with a `OMGSocketClient.Builder` before using it.
+
+```kotlin
+val socketClient = OMGSocketClient.Builder {
+    baseURL = "wss://your_ewallet_base_url/api/socket/"
+    apiKey = YOUR_API_KEY
+    authenticationToken = YOUR_AUTH_TOKEN
+    debug = true
+}.build()
```

Comment:
Just a thought, wouldn't it be nice to have an extension method to build this from an existing `OMGAPIClient` ? Since the parameters are the same 🤔 
Or maybe the "config" part could be externalized in a `APIConfig`-like class. But still exposed the same way in the builders.
Just some thoughts 😄

Reply:
I've done refactor this 👉 https://github.com/omisego/android-sdk/pull/24/commits/859b7b894a67115d77c66cb5ca57c5cd428c4970